distributionviewer/core/static/js/app/components/views/chart.js

```diff
@@ -57,18 +60,14 @@ function generateChart(name, chart, width, height) {
 
 export class Chart extends React.Component {
   componentDidMount() {
-    if (this.props.isDataReady) {
-      //generateChart(this.props.chartName, this.props.item, this.props.width, this.props.height);
-    }
+    // TODO: We need to do more here about managing isFetching.
+    axios.get(metricApi.endpoints.GET_METRIC + this.props.chartName).then(response => {
+      generateChart(this.props.chartName, response.data, this.props.width, this.props.height);
+    });
   }
```

Comment:
I guess I'm not sure what practical difference it makes, but since this renders the chart, should it be in `render`?